### PR TITLE
🐳  new stranded version

### PR DESCRIPTION
**Pull request by dmiller15**
## Description

New version of stranded that includes the following changes: https://github.com/zoomzoom1011/how_are_we_stranded_here/pull/1

Part of https://github.com/d3b-center/bixu-tracker/issues/1871

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Built locally

**Test Configuration**:
* Environment:
* Test files:

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings
- [x] I have committed any related changes to the PR

### CODE REVIEW

File: stranded/1.1.0/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+RUN apt update && apt install -y curl python-is-python3 python3-pip git
+
+WORKDIR /opt
+ENV RSEQC_VERSION 5.0.1
+RUN curl -sL https://sourceforge.net/projects/rseqc/files/RSeQC-${RSEQC_VERSION}.tar.gz/download | tar -xz && python -m pip install /opt/RSeQC-${RSEQC_VERSION}
+ENV PATH $PATH:/opt/RSeQC-${RSEQC_VERSION}/scripts
+
+RUN python -m pip install --upgrade pip \
+    && python -m pip install pandas \
+    && python -m pip install --no-deps git+https://github.com/zoomzoom1011/how_are_we_stranded_here.git
```
Comment (migbro) [inline stranded/1.1.0/Dockerfile:12]: Should we make an institutional fork for this?

**Review by migbro — APPROVED** [review]
This is the docker for auto-determining strandedness, right? It's an update to that?

File: stranded/1.1.0/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,18 @@
+FROM staphb/samtools:1.16.1
```
Comment (sakshamphul) [inline stranded/1.1.0/Dockerfile:1]: any reason why this is not at 1.19 (https://github.com/samtools/samtools/releases/) and using a lower version of samtools ?

**Review by sakshamphul — COMMENTED** [review]
a comment around samtool version. Why base image is not at 1.19 or 1.18  ?

File: stranded/1.1.0/Dockerfile (hunk 1 above)
Comment (dmiller15) [reply stranded/1.1.0/Dockerfile:12]: Probably.

File: stranded/1.1.0/Dockerfile (hunk 2 above)
Comment (dmiller15) [reply stranded/1.1.0/Dockerfile:1]: I was just trying to make minimal changes to the 1.0.0 docker. I can bump those changes.

**Comment by dmiller15** [comment]
> This is the docker for auto-determining strandedness, right? It's an update to that?

@migbro Yes this is that script. I modified the repo to print out read length median and stddev values. Bumping the version to get those changes into the docker

**Review by sakshamphul — APPROVED** [review]
Perfect!

**Pull request merged**